### PR TITLE
fix new GraphConv methods' arguments bug.

### PR DESCRIPTION
**Pull request by ktaneishi**
dc.nn.GraphConv(), dc.models.MultitaskGraphClassifier() and  dc.nn.Dense() methods needed some more arguments like n_feat. I confirmed this fix with `python examples/benchmark.py -d sampl`.

### CODE REVIEW

File: examples/benchmark.py [hunk 1]
```diff
@@ -498,22 +498,22 @@ def benchmark_classification(train_dataset,
     with g.as_default():
```
Comment (rbharath) [inline examples/benchmark.py]: We can remove this line (no longer need to keep external graphs).

File: examples/benchmark.py [hunk 2]
```diff
@@ -696,22 +696,22 @@ def benchmark_regression(train_dataset,
...
```
Comment (rbharath) [inline examples/benchmark.py]: Same here.

**Comment by rbharath** [comment]
@ktaneishi Thanks for the contribution! I'd be glad to accept the PR once the yapf bugs are fixed (just run `yapf -i examples/benchmark.py` after install `yapf` with `pip install yapf`).

I've suggested a couple lines that can be removed as well.

**Comment by ktaneishi** [comment]
Thank you for your review. I pushed your suggestion and yapf changes.

**Pull request merged**